app/javascript/app/components/ndcs-country-accordion/ndcs-country-accordion-selectors.js

```diff
@@ -0,0 +1,183 @@
+  }
+);
+
+export const getCategoriesWithSectors = createSelector(
+  [ groupIndicatorsByCategory ],
+  categories => {
+    if (!categories) return null;
+    return categories.map(cat => {
+      const sectorIds = [];
+      cat.indicators.forEach(ind => {
+        Object.keys(ind.locations).forEach(location => {
+          ind.locations[location].forEach(
+            value => value.sector_id && sectorIds.push(value.sector_id)
+          );
+        });
+      });
+      return { ...cat, sectors: sectorIds.length ? uniq(sectorIds) : null };
+    });
+  }
+);
+
+export const parsedCategoriesWithSectors = createSelector(
+  [ getCategoriesWithSectors, getSectors ],
+  (categories, sectors) => {
+    if (!categories) return null;
+    return categories.map(cat => {
+      const sectorsParsed = sortBy(
+        cat.sectors && cat.sectors.length && cat.sectors.map(sec => {
+            const definitions = [];
+            cat.indicators.forEach(ind => {
+              const valueObject = ind.locations[COUNTRY_ISO]
+                ? ind.locations[COUNTRY_ISO].find(v => v.sector_id === sec)
+                : null;
+              const value = valueObject && valueObject.value ||
+                (isNaN(parseInt(COUNTRY_ISO, 10)) ? '-' : null);
+              const descriptions = [ { iso: COUNTRY_ISO, value } ];
+              definitions.push({
+                title: ind.name,
+                slug: ind.slug,
+                descriptions
+              });
+            });
+            const parent = sectors[sec].parent_id &&
+              sectors[sectors[sec].parent_id];
+            return {
+              title: sectors[sec].name,
+              slug: snakeCase(sectors[sec].name),
+              parent,
+              definitions
+            };
+          }),
+        [ 'parent.name', 'title' ]
+      );
+      return { title: cat.name, slug: cat.slug, sectors: sectorsParsed };
+    });
+  }
+);
+
+export const getNDCs = createSelector([ getCategories, parseIndicatorsDefs ], (
+  categories,
+  indicators
+) =>
+  {
+    if (!categories) return null;
+    if (!indicators) return null;
+    const ndcs = Object
+      .keys(categories)
+      .map(category => ({
+        title: categories[category].name,
+        slug: categories[category].slug,
+        definitions: indicators[category] ? indicators[category] : []
+      }));
+    return ndcs;
+  });
+
+export const filterNDCs = createSelector([ getNDCs, getSearch ], (
+  ndcs,
+  search
+) =>
+  {
+    if (!ndcs) return null;
+    const filteredNDCs = ndcs.map(ndc => {
+      const defs = !search
+        ? ndc.definitions
+        : ndc.definitions.filter(
+          def =>
+            upperDeburr(def.title).indexOf(search) > -1 ||
+              upperDeburr(def.descriptions[0].value).indexOf(search) > -1
+        );
+
+      return { ...ndc, definitions: defs };
+    });
+    const reducedNDCs = filteredNDCs.filter(ndc => ndc.definitions.length > 0);
```

Comment:
the same story with `> 0`

Reply:
I'm used to this as a standard (or length !== 0) as sometimes returning a non-boolean can create some bugs